epub33/core/vocab/meta-property.html

```diff
@@ -796,7 +763,7 @@ <h5>Examples</h5>
 				<pre>&lt;metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
     
     &lt;dc:identifier id="pub-id">urn:uuid:A1B0D67E-2E81-4DF5-9E67-A64CBE366809&lt;/dc:identifier>
-    &lt;meta refines="#pub-id" property="identifier-type" scheme="xsd:string">uuid&lt;/meta>
+    &lt;meta refines="#pub-id" property="identifier-type">uuid&lt;/meta>
     
```

Comment:
Didn't you remove this? Or this is an issue with many PR-s?

Reply:
Ya, too many PRs going today. :)

I keep basing them off the current master branch, so fixes will repeat until I merge up again.